docs/tasks.md

```diff
@@ -21,14 +21,20 @@ The built in documentation for the task can be seen by running the following:
 ```
 bolt task show gplt::create_control_repo_from_production_env
 
-gplt::create_control_repo_from_production_env - Creates a git repo from puppet production environment directory
+gplt::create_control_repo_from_production_env - Creates a local git repo for puppet code
 
 USAGE:
-bolt task run --nodes <node-name> gplt::create_control_repo_from_production_env ctrl_repo_path=<value>
+bolt task run --nodes <node-name> gplt::create_control_repo_from_production_env ctrl_repo_path=<value> code_source_path=<value> code_source_url=<value> code_source_branch=<value>
```

Comment:
Should this have separate usage examples for `code_source_path` and `code_source_url`?

Reply:
That is bolt generated.  We have no control, and can't add anything to it.